app/actions/app/server.ts

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Navigation} from 'react-native-navigation';
+
+import {doPing} from '@actions/remote/general';
+import {fetchConfigAndLicense} from '@actions/remote/systems';
+import {Screens} from '@constants';
+import DatabaseManager from '@database/manager';
+import SecurityManager from '@managers/security_manager';
+import WebsocketManager from '@managers/websocket_manager';
+import {getServer, getServerByIdentifier, queryAllActiveServers} from '@queries/app/servers';
+import {getConfig} from '@queries/servers/system';
+import {logError} from '@utils/log';
+import {canReceiveNotifications} from '@utils/push_proxy';
+import {alertServerAlreadyConnected, alertServerError, loginToServer} from '@utils/server';
+
+import type {IntlShape} from 'react-intl';
+
+export async function initializeSecurityManager() {
+    const servers = await queryAllActiveServers()?.fetch();
+    if (!servers?.length) {
+        return;
+    }
+
+    const promises: Array<Promise<[string, ClientConfig | null]>> = [];
+    const results: Record<string, ClientConfig> = {};
+
+    for (const server of servers) {
+        try {
+            const {database} = DatabaseManager.getServerDatabaseAndOperator(server.url);
+            const promise = getConfig(database).then((config) => [server.url, config] as [string, ClientConfig | null]);
+            promises.push(promise);
+        } catch (error) {
+            logError('initializeSecurityManager', error);
+            continue;
```

Comment:
Would this allow the app to bypass security restrictions because it faced some arbitrary issue in obtaining database or config?

Reply:
And in that case would I able to access the server?